### ISSUE TITLE
Stripe module, support for Stripe payment gateway

### ISSUE THREAD

**Issue by arnisjuraga** [copona/copona#61]
Let's create?

https://github.com/opencart/opencart/issues/5182

**Comment by prhost** [comment]
Free
https://www.opencart.com/index.php?route=marketplace/extension/info&extension_id=27869&filter_search=stripe

**Comment by arnisjuraga** [comment]
Yes, just to be sure to support it on Copona. As we dont have extension repo yet.

**Comment by lucasjkr** [comment]
IF the payment process in Copona changes significantly, here is Stripes API documentation:

https://stripe.com/docs/elements

**Comment by arnisjuraga** [comment]
As soon as someone from our team will be using Stripe in a project, we can start to support it in a core.

**Comment by stefanhuber** [comment]
I have a private oc extension. I will put it on github soon. Maybe it is compatible...

**Comment by arnisjuraga** [comment]
If you don't mind, we can integrate it.
I don't have Stripe account to test. But definately can integrate together.

**Comment by stefanhuber** [comment]
Ok, I have published my opencart stripe module on git: https://github.com/beyondit/opencart-stripe-extension. This is also installable via composer. Maybe this can be the first integration i can provide for copona...

**Comment by arnisjuraga** [comment]
Great, @stefanhuber please, advice the best way, how to integrate it. Should we add this as Composer module to the new ```extensions/```structure, or just to integrate source files directly into Copona?

Opencart/Composer style will not fit Copona sturcture, as we don't have ```/upload``` folder.
